README.md

```diff
@@ -52,6 +52,26 @@ or download as zip.
 
 # How to use
 
+## Inside python code
+
+### View json file
+```
+import pyjsonviewer
+
+pyjsonviewer.view_data(json_file="dat/list.json")
+
```

Comment:
Just a question. What is a possible use case? Why do you want to call this module from your code?

Reply:
I am an IR researcher. It requires me to frequently compare older results with new results and view them conveniently.

Currently I had to log results and copy them in online json-viewers or save them in a file and view it in the editor.

Being able to invoke `pyjsonviewer` from code helps me quickly check results without the above hassle.

my [long term goal](https://github.com/Ankush-Chander/irlab) is to have a fullfldeged Desktop app that makes this comparing results, producing reports, disseminating the analysis report super easy for IR researchers.